.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,27 @@
+name: PyPI Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
We should narrow this regex only to digit patterns.
I think this should work: `"v[0-9]+.[0-9]+"`

Reply:
Yes, makes sense, perfect, I'll update it